internal/cmd/run_test.go

```diff
@@ -24,53 +26,50 @@ import (
 	"github.com/tetratelabs/func-e/internal/version"
 )
 
+func init() {
+	// Don't let urfave quit the current test process on cancel!
+	cli.OsExiter = func(code int) { log.Printf("urfave called exit: %d", code) }
```

Comment:
I'm sure @mathetake and @anuraaga could guess the rage level when I finally figured out what was quitting the whole test process.